pkg/helm/helm.go

```diff
@@ -84,6 +86,7 @@ type LoadChartConfig struct {
 	CertFile string `yaml:"certFile,omitempty"`
 	KeyFile  string `yaml:"keyFile,omitempty"`
 	CaFile   string `yaml:"caFile,omitempty"`
+	LockFile string `yaml:"lockFile,omitempty"`
```

Comment:
Actually, why specify the lock file explicitly when a helm Chart.yaml or requirements.yaml does not?